docs/features/vendored-dependencies.md

```diff
@@ -75,6 +77,32 @@ We also support json-formatted dependencies:
   ]
 }
 ```
+
+## Vendored Dependency Names and Scope
+
+The name of a vendored dependency is scoped to an organization.
+
+This means that if two different projects in your organization have the same name and version, they will be treated as the same dependency by FOSSA. The one that was scanned first will be used by FOSSA. Here's an example.
```

Comment:
You should make this example a subsection of the "Vendored Dependency..." section so that it is easier to link if other users have the same issue.

I think something like the following would be a simple change.
```
### Conflicts with dependency names
Problem
Example
```

Reply:
I think that's a good idea in general, but that makes the parent section pretty much empty in this case. I'm just going to leave it as is, as I don't think there's much benefit to pulling it out